crates/router/src/connector/multisafepay/transformers.rs

```diff
@@ -181,6 +185,46 @@ pub enum WalletInfo {
     GooglePay(GpayInfo),
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize)]
```

Comment:
Are these all derives required? If not please remove the redundant ones from all the enums and structs. `#[derive(Debug, Serialize, Clone)]` should work for requests and `#[derive(Debug, Deserialize, Clone)]`  should work for response.

Reply:
For `BankRedirectInfo`, `Eq` and `PartialEq` will be used for comparing different bank's Info that is why `Eq` is needed. 

For others, It is not required.

Reply:
Can you please point to the code where you are comparing different bank's info?

Reply:
@SamraatBansal on line no. 760.
![Uploading image.png…]()

Reply:
<img width="872" alt="image" src="https://github.com/juspay/hyperswitch/assets/52617262/9e363db8-ba04-446b-9c3f-8cea96069bfd">